src/templates/index.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'base.html' %}
```

Comment:
This looks great, but let's make it another task, see [comment](https://github.com/EugeneTurkin/forum123/pull/14/files#r1063904263)